### PR TITLE
Using bee-stack to run the bee-api dependencies

### PR DESCRIPTION
**Pull request by aevo98765**
After a discussion on the bee-community slack channel today (13th November 2024) I have added instructions that were provide by @jezekra1 on how to run the dependencies for the bee-api using the bee-stack repo. I think this could be useful for other users.

### CODE REVIEW

**Comment by aevo98765** [comment]
@jezekra1 could you take a look when you get chance! Cheers.

File: README.md [hunk 1]
```diff
@@ -58,16 +58,36 @@ module ---> dto
...
-#### MongoDB
+The easiest way to run all the dependencies for the bee-api is to:
+
+- Clone the bee-stack repository https://github.com/i-am-bee/bee-stack
+- Navigate to the bee-stack directory.
+- Run the infrastructure:
+
+```
+./bee-stack.sh clean
+./bee-stack.sh:infra
```
Comment (jezekra1) [inline README.md:71]: `./bee-stack.sh start:infra`

File: README.md [hunk 2]
```diff
@@ -58,16 +58,36 @@ module ---> dto
...
 ```
 
-#### Redis
+- Run the database seeder:
+
+```
+pnpm mikro-orm seeder:run
+```
+
+- Add values the env vars: CRYPTO_CIPHER_KEY, LLM_BACKEND, EMBEDDING_BACKEND.
```
Comment (jezekra1) [inline README.md:87]: don't forget to copy `.env.example` to `.env`

**Review by jezekra1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aevo98765) [reply README.md]: Done

**Review by xjacka — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -58,6 +58,46 @@ module ---> dto
...
+- Install packages:
+
+```
+pnpm install
+```
+
+- Run the database seeder:
+
+```
+pnpm mikro-orm seeder:run
```
Comment (xjacka) [inline README.md:85]: Only for the first time run seeder and in other cases run migrations.

Comment (xjacka) [reply README.md:85]: Migrations are introduced here: https://github.com/i-am-bee/bee-api/pull/87/files#diff-9a5696f5d0097ae95b8ca91ce055d3a33927d496192ac14c5aa3a5da6c8be9dc

File: README.md [hunk 4]
```diff
@@ -58,6 +58,46 @@ module ---> dto
...
+```
+
+- copy .env.example to .env
+
+```
+cp .env.example .env
+```
+
+- Add values to the env vars: CRYPTO_CIPHER_KEY, LLM_BACKEND, EMBEDDING_BACKEND.
```
Comment (markstur) [inline README.md:94]: and backend details

File: README.md (hunk 3 above)
Comment (aevo98765) [reply README.md:84]: Changed to address your first comment. How do the migrations impact this step?

File: README.md (hunk 4 above)
Comment (aevo98765) [reply README.md]: Done.

**Comment by aevo98765** [comment]
Hey @jezekra1, good on this one now?

**Review by jezekra1 — APPROVED** [review]

**Comment by aevo98765** [comment]
@jezekra1 can you approve the awaiting workflow and the I will merge this in if I can. Cheers

**Pull request merged**